jobs/nfstestserver/spec

```diff
@@ -11,4 +11,6 @@ properties:
  nfstestserver.export_cidr:
     description: "cidr range to use for nfs exports"
     default: "10.0.0.0/8"
-
+ nfstestserver.export_root_path:
+    description: "NFS root path"
+    default: "/var/vcap/store"
```

Comment:
The convention used in other releases is to add the job name to the path, e.g. ``/var/vcap/store/mysql`` in https://github.com/cloudfoundry/cf-mysql-release/blob/develop/jobs/mysql/templates/my.cnf.erb#L67

The default value might then benefit from being ``/var/vcap/store/nfstestserver``